automobile/lidar-sensors.md

```diff
@@ -39,7 +39,7 @@ to speed up the simulation.
 The `Sick Lms 291` is a 1 layer lidar with a range of up to 80 meters and a
 field of view of up to 180 degrees.
 
-The model of the `IbeoLux` contains a spherical projection, a configurable fixed
+The model of the `Sick Lms 291` contains a spherical projection, a configurable fixed
```

Comment:
Not too sure about this one, it seemed odd to mention the previous LIDAR in the `Sick Lms 291` section ...